src/theme/stories/index.stories.js

```diff
@@ -4,8 +4,8 @@ import Box from 'ui-box'
 import Component from '@reactions/component'
 import { Heading } from '../../typography'
 import { Checkbox } from '../../checkbox'
-import v5Theme from '../src/themes/v5'
-import v6Theme from '../src/themes/default'
+import v5Theme from '../../themes/v5'
```

Comment:
How about updating this to themes/classic?

Reply:
Can do!